src/Chapter2.hs

```diff
@@ -783,9 +803,15 @@ divideTenBy :: Int -> Int
 divideTenBy x = div 10 x
 
 -- TODO: type ;)
+listElementsLessThan :: Int -> [Int] -> [Int]
 listElementsLessThan x l = filter (< x) l
 
 -- Can you eta-reduce this one???
+{- 
+No, I can't :(( Maybe I can only create a function that takes the first list and returns a new one, 
+which takes the second list and multiplies the lists element by element. 
+-}
+pairMul :: [Int] -> [Int] -> [Int]
```

Comment:
I tried to explain eta-reduction in my other comment. Let us know if you have more questions :slightly_smiling_face: 

I hope it will become clearer, and you will be able to crack this task :muscle: 
Btw, nice job on specifying type signatures for these functions! The types are correct :ok_hand: